frontend/www/js/omegaup/components/qualitynomination/List.vue

```diff
@@ -1,94 +1,97 @@
+            <th>{{ T.wordsAlias }}</th>
+            <th>{{ T.wordsNominator }}</th>
+            <th>{{ T.wordsAuthor }}</th>
+            <th>{{ T.wordsSubmissionDate }}</th>
+            <th class="text-center">{{ T.wordsStatus }}</th>
+            <th><!-- view button --></th>
+          </tr>
+        </thead>
+        <tbody>
+          <tr v-for="nomination in visibleNominations">
+            <td class="text-center">{{ nomination.nomination }}</td>
+            <td>
+              <a v-bind:href="problemUrl(nomination.problem.alias)">{{
+                nomination.problem.title
+              }}</a>
+            </td>
+            <td>
+              <a v-bind:href="userUrl(nomination.nominator.username)">{{
+                nomination.nominator.username
+              }}</a>
+            </td>
+            <td>
+              <a v-bind:href="userUrl(nomination.author.username)">{{
+                nomination.author.username
+              }}</a>
+            </td>
+            <td>{{ nomination.time.format('long') }}</td>
+            <td class="text-center">{{ nomination.status }}</td>
+            <td>
+              <a
+                v-bind:href="
+                  nominationDetailsUrl(nomination.qualitynomination_id)
+                "
+                >{{ T.wordsDetails }}</a
+              >
+            </td>
+          </tr>
+        </tbody>
+      </table>
     </div>
-    <table class="table table-striped">
-      <thead>
-        <tr>
-          <th class="text-center">{{ T.qualityNominationType }}</th>
-          <th>{{ T.wordsAlias }}</th>
-          <th>{{ T.wordsNominator }}</th>
-          <th>{{ T.wordsAuthor }}</th>
-          <th>{{ T.wordsSubmissionDate }}</th>
-          <th class="text-center">{{ T.wordsStatus }}</th>
-          <th><!-- view button --></th>
-        </tr>
-      </thead>
-      <tbody>
-        <tr v-for="nomination in visibleNominations">
-          <td class="text-center">{{ nomination.nomination }}</td>
-          <td>
-            <a v-bind:href="problemUrl(nomination.problem.alias)">{{
-              nomination.problem.title
-            }}</a>
-          </td>
-          <td>
-            <a v-bind:href="userUrl(nomination.nominator.username)">{{
-              nomination.nominator.username
-            }}</a>
-          </td>
-          <td>
-            <a v-bind:href="userUrl(nomination.author.username)">{{
-              nomination.author.username
-            }}</a>
-          </td>
-          <td>{{ nomination.time.format('long') }}</td>
-          <td class="text-center">{{ nomination.status }}</td>
-          <td>
-            <a
-              v-bind:href="
-                nominationDetailsUrl(nomination.qualitynomination_id)
-              "
-              >{{ T.wordsDetails }}</a
-            >
-          </td>
-        </tr>
-      </tbody>
-    </table>
+    <omegaup-common-paginator
+      v-bind:pagerItems="pagerItems"
```

Comment:
```suggestion
      v-bind:pager-items="pagerItems"
```

Vue sabe que las cosas en kebab-case corresponden con nombres en camelCase! entonces esto de cualquier manera va a asignar a la propiedad llamada `pagerItems`.